clients/recreation_client.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+import requests
+from fake_useragent import UserAgent
+
+from utils import formatter
+
+LOG = logging.getLogger(__name__)
+
+
+class RecreationClient:
+
+    BASE_URL = "https://www.recreation.gov"
+    AVAILABILITY_ENDPOINT = BASE_URL + "/api/camps/availability/campground/{park_id}/month"
+    MAIN_PAGE_ENDPOINT = BASE_URL + "/api/camps/campgrounds/{park_id}"
+
+    headers = {"User-Agent": UserAgent(verify_ssl=False).random}
+
+    @staticmethod
```

Comment:
Let's make this a classmethod instead and use `cls.AVAILABILITY_ENDPOINT`, `cls._send_request`, etc.

Reply:
Done.